lib/puck/jar.rb

```diff
@@ -91,12 +99,21 @@ def create
         temporary_output_path = File.join(Dir.mktmpdir, @configuration[:jar_name])
         project_dir = Pathname.new(@configuration[:app_dir]).expand_path.cleanpath
         extra_files = @configuration[:extra_files] || []
-        jruby_complete_path = @configuration[:jruby_complete]
+        merge_archives = @configuration[:merge_archives] || []
+        jruby_complete_path = @configuration[:jruby_complete] || [JRubyJars.core_jar_path, JRubyJars.stdlib_jar_path]
+        jruby_complete_path = [jruby_complete_path].flatten
 
         if !(defined? JRubyJars) && !(jruby_complete_path && File.exists?(jruby_complete_path))
           raise PuckError, 'Cannot build Jar: jruby-jars must be installed, or :jruby_complete must be specified'
         end
 
+        archives = merge_archives.dup
+        if archives.is_a?(Hash)
+          jruby_complete_path.each_with_object(archives) {|archive, object| object[archive] = archive}
```

Comment:
I think this will put the jar in the wrong place, if I specify `:jruby_complete => 'foo.jar'` and it contains `bar.class` that class will end up as `foo.jar/bar.class` in the Puck jar.